k8s-ingress/ingress-controller.yaml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: l5d-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: l5d-config
+  namespace: l5d-system
+data:
+  config.yaml: |-
+    admin:
+      port: 9990
+
+    namers:
+    - kind: io.l5d.k8s
+
+    routers:
+    - protocol: http
+      identifier:
+        kind: io.l5d.http.ingress
+      servers:
+        - port: 80
+          ip: 0.0.0.0
+          clearContext: true
+      dtab: /svc => /#/io.l5d.k8s
+
+    usage:
+      orgId: linkerd-examples-ingress
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: l5d
+  namespace: l5d-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: l5d
+    spec:
+      volumes:
+      - name: l5d-config
+        configMap:
+          name: "l5d-config"
+      containers:
+      - name: l5d
+        image: buoyantio/linkerd:0.9.1
+        env:
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
+        args:
+        - /io.buoyant/linkerd/config/config.yaml
+        ports:
+        - name: http
+          containerPort: 80
+          hostPort: 80
+        - name: tls
+          containerPort: 443
+        - name: admin
+          containerPort: 9990
+        volumeMounts:
+        - name: "l5d-config"
+          mountPath: "/io.buoyant/linkerd/config"
+          readOnly: true
+
+      - name: kubectl
+        image: buoyantio/kubectl:v1.4.0
+        args: ["proxy", "-p", "8001"]
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: l5d
+  namespace: l5d-system
+spec:
+  selector:
+    app: l5d
+  type: NodePort
+  ports:
+  - name: http
+    port: 80
+  - name: tls
+    port: 443
+  - name: admin
+    port: 9990
```

Comment:
Hmm, I wasn't able to get linkerd's admin interface to successfully load externally on port 9990. Not sure what the issue is.